Mathlib/Data/Nat/Multiplicity.lean

```diff
@@ -121,6 +122,14 @@ theorem multiplicity_factorial {p : ℕ} (hp : p.Prime) :
         congr_arg _ <| Finset.sum_congr rfl fun _ _ => (succ_div _ _).symm
 #align nat.prime.multiplicity_factorial Nat.Prime.multiplicity_factorial
 
+ theorem sub_one_mul_multiplicity_factorial_eq_sub_sum_digits {p : ℕ} (hp : p.Prime) :
```

Comment:
I'm inclined to shorten the name to
```suggestion
 theorem sub_one_mul_multiplicity_factorial {p : ℕ} (hp : p.Prime) :
```
Could you please also add a docstring with the math formula, because in this case the readability of the statement is somewhat hindered by the rather long argument to `.get`.

If you end up shortening the name, please update the title of the PR on github.

Reply:
I 
 - shortened the name
 -  also in the title and in the doc string
 - also shortened the analogues title for `padicValNat`
 
 Added a docstring, which constains some formulae in a sentence, but not exactly a latex math-formula, because both "multiplicity" and "sum of digits" don't seem to have good notation, so it seems just more natural writing a sentence than a formula, I hope that's ok like this.